eslint.config.mjs

```diff
@@ -0,0 +1,81 @@
+import jqueryConfig from "eslint-config-jquery";
+import globals from "globals";
+
+export default [
+	{
+		linterOptions: {
+			reportUnusedDisableDirectives: "error"
+		}
+	},
+
+	{
+		ignores: [
+			"app/dist/**",
```

Comment:
app/dist was not part of the former .eslintignore file. Is this on purpose?

Reply:
Those files never passed ESLint. It's just that our Grunt ESLint config is:
```js
	eslint: {
		all: [ "*.js", "test/*js", "lib/**/*.js", "app/src/*.js" ]
	},
```
so those files were never checked anyway. But if you'd run `npx eslint .`, you'd get lots of errors. Now that command is green as well - plus IDEs don't pass dist files through ESLint when viewing.

Reply:
Ahh, perfect. Just wanted to be sure.